patterns/stories-podcast-banner.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Title: Stories Podcast Hero
+ * Slug: twentytwentyfive/stories-podcast-hero
+ * Categories: featured, hero
+ * Keywords: podcast, hero, stories
+ *
+ * @package TwentyTwentyFive
```

Comment:
Please update the package information to this format

 * @package WordPress
 * @subpackage Twenty_Twenty_Five
 * @since Twenty Twenty-Five 1.0
 * 
 https://github.com/WordPress/twentytwentyfive/issues/98

Reply:
Hey @carolinan 

Can do making that change now! 

is there a documentation piece you can point to for why this would be so I can know for next pattern I work on? Just a bit curious this hasn't come up in some of the resources I read? 
Referring to my notes from https://developer.wordpress.org/themes/patterns/registering-patterns/ 

Thanks for the feedback :)